src/ElementHandle.re

```diff
@@ -49,3 +52,6 @@ external type_ :
 
 [@bs.send.pipe: t] [@bs.splice]
 external uploadFile : (~filePaths: array(string)) => Js.Promise.t(unit) = "";
+
+[@bs.send.pipe: t]
+external contentFrame : Js.Promise.t(Js.Nullable.t(Types.frameBase)) = "";
```

Comment:
Should likely also be Js.null. We can use Js.nullToOption etc.

Reply:
Fixed in commit 51cb1c3